docs/Changelog.md

```diff
@@ -43,9 +43,11 @@
   * toBigIntegerOrNull
   * toBigDecimal
   * toBigDecimalOrNull
+* [#6995](https://github.com/TouK/nussknacker/pull/6995) Add `toJson` and `toJsonString` conversions
```

Comment:
Where these conversions are available? :) I mean from which helper I can rich it?